coalib/coala_json.py

```diff
@@ -45,6 +46,27 @@ def main():
                 results.append(bear)
         except BaseException as exception:  # pylint: disable=broad-except
             return get_exitcode(exception, log_printer)
+    elif args.show_capabilities:
+        local_bears, global_bears = get_filtered_bears(
+            args.filter_by_language, log_printer)
+        capabilities = filter_capabilities_by_languages(
+            local_bears, args.show_capabilities)
+        retval = {"capabilities Can detect/Can fix": capabilities}
+        if args.output:
+            filename = str(args.output[0])
+            with open(filename, 'w+') as fp:
+                json.dump(retval, fp,
+                          cls=JSONEncoder,
+                          sort_keys=True,
+                          indent=2,
+                          separators=(',', ': '))
+        else:
+            print(json.dumps(retval,
+                             cls=JSONEncoder,
+                             sort_keys=True,
+                             indent=2,
+                             separators=(',', ': ')))
```

Comment:
I dislike duplicating the code here, can we work around that? Feel free to put the other commit in an own PR, reack and merge it.